integrations/sarama/sarama_kakfa.go

```diff
@@ -44,6 +44,23 @@ func (m *saramaConsumerWrapper) GetTopic() *string { return &m.ConsumerMessage.T
 
 func (m *saramaConsumerWrapper) GetPartition() int32 { return m.ConsumerMessage.Partition }
 
+func (m *saramaProducerWrapper) GetSize() int64 {
+	var headersSize = 0
+	for _, header := range m.Headers {
+		headersSize += len(header.Key) + len(header.Value)
+	}
+	return int64(m.Key.Length()) + int64(m.Value.Length())
```

Comment:
headersSize is not being used in this returned sum, whereas on line 61 it is being used but the comment says ignoring header sizes for now